### PR TITLE
Error when sending an email using EmailService

### PR DESCRIPTION
**Pull request by synbot**
## Error when sending an email using EmailService

 PHP Fatal error:  Call to undefined method Synapse\Email\SendEmailCommand::run() in /vagrant/vendor/synapsestudios/synapse-base/src/Synapse/Work/AbstractConsoleWork.php on line 39

`run` is called, but the method that needs to run is called `execute`.

### CODE REVIEW

**Comment by zpchavez** [comment]
:o:

**Comment by zpchavez** [comment]
:star:

**Comment by paulstatezny** [comment]
:pray:

File: src/Synapse/Email/SendEmailCommand.php [hunk 1]
```diff
@@ -3,13 +3,15 @@
 namespace Synapse\Email;
 
 use Synapse\Command\CommandInterface;
```
Comment (paulstatezny) [inline src/Synapse/Email/SendEmailCommand.php]: This can be removed.

**Comment by paulstatezny** [comment]
:+1:

**Pull request merged**